iteration/iteration.go

```diff
@@ -26,19 +26,25 @@ const (
 	IterationStateClose    = "close"
 	PathSepInService       = "/"
 	PathSepInDatabase      = "."
+	IterationActive        = bool(true)
+	IterationNotActive     = bool(false)
 )
 
+var active bool
```

Comment:
I dont need that variable at all with the new changes that I implemented. So, removed the variable -> a9127c0b7b079a0dc2ec07e20776ad4a354d6e53